resources/lemmas.md

```diff
@@ -72,6 +83,13 @@ They capture the essential mechanisms used by the two instructions: splitting a
                 : nthbyteof(V, 31, 32)
                 : .WordStack ) => V
       requires 0 <=Int V andBool V <Int pow256
+
+    rule 0 <=Int #asWord(#bufSeg(_, _, _))                 => true
+    rule         #asWord(#bufSeg(_, _, WIDTH)) <Int pow256 => true  requires WIDTH <=Int 32
```

Comment:
if we want to allow #asWord take more than 32 bytes, the `requires` is not needed i think.

Reply:
agreed